File: sib/src/main/java/com/ljwx/sib/common/src/main/java/com/sisensing/common/base/TrustAllHostnameVerifier.java
package com.sisensing.common.base;

import javax.net.ssl.HostnameVerifier;
import javax.net.ssl.SSLSession;

class TrustAllHostnameVerifier implements HostnameVerifier {
    @Override
    public boolean verify(String hostname, SSLSession session) {
        return true;
    }
}

File: baseapp/src/main/java/com/ljwx/baseapp/net/ssl/TrustAllHostnameVerifier.kt
package com.ljwx.baseapp.net.ssl

import javax.net.ssl.HostnameVerifier
import javax.net.ssl.SSLSession

class TrustAllHostnameVerifier : HostnameVerifier {
    override fun verify(hostname: String?, session: SSLSession?): Boolean {
        return true
    }
}